Forge/Server/Data/LiteDbCharacterTagService.cs

```diff
@@ -40,6 +40,20 @@ public bool Update(CharacterTagModel tag)
                 .Update(tag);
         }
 
+        public bool DeleteOne(Guid id)
+        {
+            CharacterTagModel tag = _liteDb.GetCollection<CharacterTagModel>("CharacterTag").Find(x => x.Id == id).FirstOrDefault();
```

Comment:
Please use the "FindOne" repository method instead of directly accessing the collection. Also use "var".